torch/_inductor/ir.py

```diff
@@ -5470,7 +5486,6 @@ def get_inputs_that_alias_output(self):
         return self.alias_names
 
     def get_mutation_names(self):
-        assert len(self.mutation_names) <= 1
```

Comment:
Based on https://github.com/pytorch/pytorch/pull/112547, we should not be returning more than one name in `get_mutation_names`. Instead we may need to go the route that triton kernels did for mutating multiple inputs? @oulgen @Chillee can you check me on this?

Reply:
Yep, inductor does not support more than 1 mutation names, you'll hit another assert in scheduler. You can do the same thing I did for triton kernels

Reply:
Changed `FallbackKernel` to use same approach as `UserDefinedTritonKernel`. This is changing existing code and will see if any unit test error

Reply:
What is lacking in the support for > 1 mutation names ? It seems to generalize to multiple mutations:

https://github.com/pytorch/pytorch/blob/main/torch/_inductor/scheduler.py#L1653-L1667